config/locales/en.yml

```diff
@@ -115,6 +115,8 @@ en:
     referee_type: What kind of referee do you want to add?
     referee_destroy: Are you sure you want to delete this referee?
     referee_cancel: Are you sure you want to cancel this request for a reference?
+    decoupled_reference_cancel: Are you sure you want to cancel this reference request?
+    reference_destroy: Are you sure you want to delete this reference request?
```

Comment:
Stray addition? Not seeing this used anywhere.

Reply:
Hmm yeah I think it must be. Thought it was in place already but must've added it experimentally. Got rid of it.